package.json

```diff
@@ -38,9 +52,19 @@
         "eslint-config-prettier": "^9.1.0",
         "eslint-plugin-react": "^7.35.0",
         "npm-license-crawler": "^0.2.1",
-        "prettier": "^3.3.3"
+        "prettier": "^3.2.5"
     },
     "dependencies": {
-        "typescript": "^5.5.3"
-    }
+        "inquirer": "^10.2.0",
+        "typescript": "^4.9.5"
+    },
+    "files": [
+        "LICENSE",
+        "README.md",
+        "CHANGELOG.md",
+        "templates",
+        "images",
+        "index",
+        "package.json"
```

Comment:
package.json is included automatically, so it's not necessary to list it in the files entry.